### ISSUE TITLE
Fatal error - Possible integer overflow in memory allocation

### ISSUE THREAD

**Issue by plsoucy-tapclicks** [snowflakedb/pdo_snowflake#153]
We've had several queries throw memory errors like this using the PDO driver in the past, while they work fine using the ODBC driver or through the Snowflake UI.

**Fatal error: Possible integer overflow in memory allocation (18446744073709551615 * 48 + 0)**

I was just able to isolate an example that doesn't require sharing too much of our code here on GitHub.

The query:

`$snowflake->runQuery("CREATE OR REPLACE TABLE obfuscated_table_name (daily_budget number(38,6))");`

The runQuery function does the following:

```
        $sf_db = $this->getPdoObject();
        $query = $sf_db->prepare($query);
        $query->execute();
```

... and the getPdoObject function does:

```
        $pdo_identifier = 'snowflake:account='.str_replace('.snowflakecomputing.com', '', self::$snowflake_config_details['server_name']);

        $sf_db = new PDO($pdo_identifier, self::$snowflake_config_details['username'], self::$snowflake_config_details['password']);
        $sf_db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
        $sf_db->query('USE '.self::$snowflake_config_details['database']);
        $sf_db->query('USE WAREHOUSE '.self::$snowflake_config_details['warehouse']);

        return $sf_db;
```

The fatal error results from the $query->execute(); line.

This is on PHP 7.0.*, with the latest version of the PDO drivers.

Can you please investigate and address this? Thanks.

**Comment by eddychansnowflake** [comment]
Pierre-Luc, our engineer has been able to reproduce the issue and is looking into it standby for further updates.

**Comment by notkriswagner** [comment]
Hi, I've found the source of the issue and am currently working on a fix.

**Comment by notkriswagner** [comment]
Related to snowflakedb/libsnowflakeclient#76

**Comment by eddychansnowflake** [comment]
@notkriswagner Want to confirm that this is fixed now as it looks like #76 is fixed?

**Comment by notkriswagner** [comment]
Fixed in PR #155
